t/10-npg_warehouse-loader-irods-location.t

```diff
@@ -116,11 +116,37 @@ my $new_same_row =
     "last_changed" => ignore(),
   };
 
+my $directory = [
+  {
+    "id_product" => "42kcie64a98b1dad5ec481b82e2465kf5a9647c537f99381093c93cd6fkd73rt",
+    "seq_platform_name" => "illumina",
+    "pipeline_name" => "npg-prod",
+    "irods_root_collection" => "/seq/illumina/20/20202/lane1/plex3/",
+    "irods_data_relative_path" => "20202_1#3.bam",
+    "irods_secondary_data_relative_path" => undef,
+    "id_seq_product_irods_locations_tmp" => ignore(),
```

Comment:
Could you delete the keys that will not be present in the file that is created by the iRODS loader?